src/components/pages/home/jumbotron/index.tsx

```diff
@@ -95,12 +95,15 @@ const Jumbotron: FunctionComponent<JumbotronProps> = ({resource}) => {
           </div>
         </div>
       </div>
-      <UniqueBackground className="absolute left-0 top-0 w-full h-full z-0 object-cover" />
+      <UniqueBackground
+        className="absolute left-0 top-0 w-full h-full z-0 object-cover"
+        background={resource.background}
```

Comment:
we would like to use SVG instead of raster in some cases, so I'm wondering whether that would be possible with Sanity?